assignment/src/models/patient.ts

```diff
@@ -0,0 +1,53 @@
+import Joi, { ValidationError } from 'joi';
+import mongoose, { Document, Schema } from 'mongoose';
+
+interface IPatient extends Document {
+    petName: string;
+    petType: 'Cat' | 'Dog' | 'Bird';
+    ownerName: string;
```

Comment:
I think we can just name these field name, address and phone as our interface already conving their names